src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8814,11 +8814,11 @@ namespace UnitarySystems {
         CoolingLoad = false;
         HeatingLoad = false;
 
-        if (QZnReq > Small5WLoad / this->ControlZoneMassFlowFrac && !DataZoneEnergyDemands::CurDeadBandOrSetback(this->ControlZoneNum)) {
+        if (QZnReq > Small5WLoad / this->ControlZoneMassFlowFrac) { // no need to check deadband flag, QZnReq is correct.
```

Comment:
@rraustad A small suggestion for future efficiency improvement (not now - might introduce tiny diffs):
`Small5WLoad/this->ControlZoneMassFlowFrac` is constant over the course of a simulation, correct? So, could compute that one upfront and store in something like `this->SmallLoadTolerance` instead of recomputing this on every pass.

Reply:
Got it.

Reply:
Done